sycl/include/sycl/detail/vector_convert.hpp

```diff
@@ -80,12 +80,9 @@ using is_uint_to_uint =
     std::bool_constant<is_sugeninteger_v<T> && is_sugeninteger_v<R>>;
 
 template <typename T, typename R>
-using is_sint_to_uint =
-    std::bool_constant<is_sigeninteger_v<T> && is_sugeninteger_v<R>>;
-
-template <typename T, typename R>
-using is_uint_to_sint =
-    std::bool_constant<is_sugeninteger_v<T> && is_sigeninteger_v<R>>;
+using is_sint_to_from_uint = std::bool_constant<
```

Comment:
I'm a bit confused about the meaning of this. Before the change, this was `is_sint_to_uint` which I understand as converting from `signed int` to `unsigned int`. With the current name, I'm not sure if it's still the same or the meaning changed. Looking at the condition, it seems that is both from `signed int` to `unsigned int` and from `unsigned int` to `signed int`. Is that right?

Reply:
Right, I'm actually reverting it back to what we had before #11770: integer conversions which change sign are now handled uniformly regardless of signs of source and destination types